ex1.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="ex1_header_css.css">
+	<link rel="stylesheet" type="text/css" href="ex2_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="buttons.css">
+</head>
+<body style="margin: 0">
+
+	<section>
+
+		<div class="header">
+			<div class="logindiv">
+				<div class="logintitle">
+					<h1>Interview Feedback</h1>
+				</div>
+				
+				<form action="/action_page.php" class="loginform">
+					<input type="text" placeholder="Username" name="name_box" class="loginboxes">
+					<br><br>
```

Comment:
Markup used for styling is evil and should go away ASAP, yes? Especially L22, which is just... yuck! :stuck_out_tongue: